src/main/java/com/google/cloud/tools/app/api/deploy/DeployConfiguration.java

```diff
@@ -28,17 +28,17 @@
 
   String getDockerBuild();
 
-  boolean isForce();
```

Comment:
On second thought since its really just a flag (it doesn't take in a value like -force true / false). Maybe what I'm saying doesn't make sense.